docs/products/postgresql/concepts/upgrade-failover.rst

```diff
@@ -58,6 +58,9 @@ During maintenance updates, cloud migrations, or plan changes, the below procedu
 .. Note::
     The old primary server is kept alive for a short period of time (minimum 60 seconds) with a TCP forwarding setup pointing to the new primary server allowing clients to connect before learning the new IP address.
 
+.. Note::
+    If the service plan is changed from a business plan that has two (2) nodes to a startup plan which only has one (1) node, the standby node will be removed while the primary node will be retained and connections to the primary are not affected by the downgrade.  Similarly, upgrading the service plan from startup to business will simply add a standby node to the service cluster and connections to the primary node are unaffected.
```

Comment:
changing from the future tense to the present tense since we're talking about a general truth here, something that is always right + a few minor changes:

 If the service plan is changed from a business plan that has two nodes to a startup plan which only has one (1) node, the standby node is removed while the primary node is retained, and connections to the primary are not affected by the downgrade.  Similarly, upgrading the service plan from a startup one to a business one simply adds a standby node to the service cluster, and connections to the primary node are unaffected.